src/web/mjs/connectors/MangaReaderTo.mjs

```diff
@@ -53,9 +53,9 @@ export default class MangaReaderTo extends Connector {
         let request = new Request(uri, this.requestOptions);
         let data = await this.fetchDOM(request, this.queryChapters);
         return data.map(element => {
-            const link = this.getRootRelativeOrAbsoluteLink(element, this.url);
-            const title = element.title.replace(/:(.*)/gi, '');
-            const lang = link.match(/(\/en\/)|(\/ja\/)|(\/ko\/)|(\/zh\/)/gi);
+            const link = this.getRootRelativeOrAbsoluteLink(element.tagName.toLowerCase() === 'a' ? element : element.querySelector('a'), this.url);
+            const title = element.tagName.toLowerCase() === 'a' ? element.title.replace(/:(.*)/gi, '') : element.textContent.trim();
```

Comment:
Is this regex safe for replacement? It looks like it may accidentally remove valid data from the title as well...

Reply:
It removes real chapter title sometimes (`Chapter 39.5: Extra: Marcus' First Week at Kings` -> `Chapter 39.5`), but most of the time chapter title looks like this: `Chapter 44: Chapter 44`.
So I think this `replace` function needs to be improved.
I suggest to add checking if the text before the first colon is the same as the text after that colon.
```js
let title;
if (element.tagName.toLowerCase() === 'a') {
    title = element.title.replace(/.*(:.*)/, (match, g1) => {
        const defaultChapterTitle = match.match(/(.*):/)[1].trim();
        return defaultChapterTitle.toLowerCase() === g1.slice(1).trim().toLowerCase() ? defaultChapterTitle : match;
    }).trim();
} else {
    title = element.textContent.trim();
}
```
But maybe this code can be simplified a bit...
And with ternary operator, I think it looks wrong, but also works.
```js
const title = element.tagName.toLowerCase() === 'a' ? element.title.replace(/.*:(.*)/, (match, g1) => {
    const defaultChapterTitle = match.match(/(.*):/)[1].trim();
    return defaultChapterTitle.toLowerCase() === g1.trim().toLowerCase() ? defaultChapterTitle : match;
}).trim() : element.textContent.trim();
```

Reply:
Tried to implement it in c9247d4